src/coreclr/System.Private.CoreLib/src/System/Reflection/RuntimeCustomAttributeData.cs

```diff
@@ -1642,13 +1642,6 @@ private static bool FilterCustomAttributeRecord(
             if (!AttributeUsageCheck(attributeType, mustBeInheritable, ref derivedAttributes))
                 return false;
 
-            // Windows Runtime attributes aren't real types - they exist to be read as metadata only, and as such
```

Comment:
This is the CoreCLR/JIT CoreLib - why is the pull request title "Remove exclusion of WindowsRuntime attribute instantation in NativeAOT"?

Reply:
I'll be honest, I initially found it through the NativeAOT file and didn't realize that this one was in CoreCLR as I searched for usages throughout the runtimes.

Reply:
Updated the title.